client/src/main/java/org/apache/falcon/ExtensionHandler.java

```diff
@@ -185,8 +185,6 @@ private static String createStagePath(String extensionName, String jobName) {
                 for (File innerFile : files) {
                     if (innerFile.isFile()) {
                         urls.add(innerFile.toURI().toURL());
```

Comment:
Don't want to support sub-directories under the build lib path? It is fine to take that stance, but, we will have to call that out explicitly in the documentation.